template/components/Logo/Logo.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { render, axe } from '../../test-utils';
+
+import { Logo } from './Logo';
+
+describe('Logo', () => {
+  /**
+   * Having a separate rendering function for your components makes it easier
+   * to render a separate component for each test and reduces boilerplate.
+   */
+  function renderLogo(props = {}) {
+    return render(<Logo {...props} />);
+  }
```

Comment:
When going with this, I would also allow for providing the testling library options to render.

```suggestion
  function renderLogo(props = {}, options = {}) {
    return render(<Logo {...props} />, options);
  }
```

Reply:
Argh, this wasn't the template. 😂

Reply:
Unfortunately, I've already merged the Foundry PR for the updated template 😬 I think it's a good point, will update it here and there.